ledger/ledger-api-tests/suites/src/main/scala/com/daml/ledger/api/testtool/suites/v1_8/TransactionServiceVisibilityIT.scala

```diff
@@ -97,6 +97,13 @@ class TransactionServiceVisibilityIT extends LedgerTestSuite {
           def go(eventId: String): Unit = {
             eventsToObserve.remove(eventId) match {
               case Some(TreeEvent(Exercised(exercisedEvent))) =>
+                val expected = exercisedEvent.childEventIds.sortBy(_.split(':')(1).toInt)
```

Comment:
Pls note string order wont work here for example: `#64:7` vs `#64:17`

Reply:
How about using `com.daml.lf.ledger.EventId.assertFromString` to parse the event ids, and then sorting by `_.nodeId`?

Reply:
> How about using com.daml.lf.ledger.EventId.assertFromString to parse the event ids, and then sorting by _.nodeId?
done